ddl/ddl.go

```diff
@@ -395,7 +400,7 @@ func (d *ddl) Start(ctxPool *pools.ResourcePool) error {
 
 		err = kv.RunInNewTxn(d.ctx, d.store, true, func(ctx context.Context, txn kv.Transaction) error {
 			t := meta.NewMeta(txn)
-			globalSeqNum, err = t.GetHistoryDDLCount()
```

Comment:
Does it add the ```Lock``` to avoid data race？ it is necessary to add by checking the stack from #32541.